api/handler/v1/org.go

```diff
@@ -18,10 +19,10 @@ import (
 )
 
 type OrganizationService interface {
-	GetOrganization(ctx context.Context, id string) (org.Organization, error)
-	CreateOrganization(ctx context.Context, org org.Organization) (org.Organization, error)
-	ListOrganizations(ctx context.Context) ([]org.Organization, error)
-	UpdateOrganization(ctx context.Context, toUpdate org.Organization) (org.Organization, error)
+	GetOrganization(ctx context.Context, id string) (modelv1.Organization, error)
```

Comment:
@krtkvrm I think we don't need to version internal models. So would suggest removing versioning from here.

Reply:
yeah.. makes sense 👍

Reply:
done